Framework/Algorithms/src/PolarizationCorrections/HeliumAnalyserEfficiency.cpp

```diff
@@ -51,9 +51,9 @@ void HeliumAnalyserEfficiency::init() {
   auto validator = std::make_shared<CompositeValidator>();
   validator->add<WorkspaceUnitValidator>("Wavelength");
   validator->add<HistogramValidator>();
-  declareProperty(
-      std::make_unique<WorkspaceProperty<>>(PropertyNames::INPUT_WORKSPACE, "", Direction::Input, validator),
-      "Input group workspace to use for polarization calculation");
+  declareProperty(std::make_unique<WorkspaceProperty<Mantid::API::WorkspaceGroup>>(PropertyNames::INPUT_WORKSPACE, "",
```

Comment:
Apologies, I can't get a comment onto the relevant line as it's not part of the code that has been changed, but lines 120-137 (the methods `validateGroupInput` and `processGroups`) can be removed now that we're setting the input workspace property type to a `WorkspaceGroup`.

By default, when a Mantid algorithm has a normal workspace input property (rather than a `WorkspaceGroup` input property) then passing a workspace group to this will loop through the group and call the algorithm against each child workspace individually. The `processGroups` method is used as a way of overriding this default behaviour when we want the algorithm to do something different with a workspace group. We needed to use this method when the algorithm was set up as it was originally, however now that we're explicitly specifying the input property to be a `WorkspaceGroup` it will work as we need it to without defining the `processGroups` method.